css/style.css

```diff
@@ -6,25 +6,140 @@ a {
   text-decoration: none;
 }
 
+.social__link {
+  transition: .5s;
+}
+
 body {
-  font-family: "Gotham Pro";
+  font-family: "Gotham Pro", sans-serif;;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)